### ISSUE TITLE
v1.10 fails during precompilation

### ISSUE THREAD

**Issue by mzy2240** [COPT-Public/COPT.jl#26]
COPT v6.5.2 is not supported?

```julia
Failed to precompile COPT [227a2e2d-e949-4d8e-a1da-7384fe6f0b9f] to /Users/zmao/.julia/compiled/v1.8/COPT/jl_uemNig.
ERROR: LoadError: You have installed version 6.5.2 of COPT, which is not supported
by COPT.jl. We require at least COPT version 4.0.5.

If you have a newer version of COPT installed, changes may need to be made
to the Julia code. Please open an issue at
https://github.com/COPT-Public/COPT.jl.
Stacktrace:
 [1] error(s::String)
   @ Base ./error.jl:35
 [2] top-level scope
   @ ~/.julia/packages/COPT/4DWVe/src/COPT.jl:93
 [3] include
   @ ./Base.jl:419 [inlined]
 [4] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt64}}, source::Nothing)
   @ Base ./loading.jl:1554
 [5] top-level scope
   @ stdin:1
in expression starting at /Users/zmao/.julia/packages/COPT/4DWVe/src/COPT.jl:7
in expression starting at stdin:1

Stacktrace:
 [1] pkgerror(msg::String)
   @ Pkg.Types /Applications/Julia-1.8.app/Contents/Resources/julia/share/julia/stdlib/v1.8/Pkg/src/Types.jl:67
 [2] precompile(ctx::Pkg.Types.Context, pkgs::Vector{String}; internal_call::Bool, strict::Bool, warn_loaded::Bool, already_instantiated::Bool, kwargs::Base.Pairs{Symbol, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
   @ Pkg.API /Applications/Julia-1.8.app/Contents/Resources/julia/share/julia/stdlib/v1.8/Pkg/src/API.jl:1432
 [3] precompile
   @ /Applications/Julia-1.8.app/Contents/Resources/julia/share/julia/stdlib/v1.8/Pkg/src/API.jl:1063 [inlined]
 [4] #precompile#225
   @ /Applications/Julia-1.8.app/Contents/Resources/julia/share/julia/stdlib/v1.8/Pkg/src/API.jl:1062 [inlined]
 [5] precompile (repeats 2 times)
   @ /Applications/Julia-1.8.app/Contents/Resources/julia/share/julia/stdlib/v1.8/Pkg/src/API.jl:1062 [inlined]
 [6] top-level scope
   @ REPL[12]:1
```

**Comment by mzy2240** [comment]
```julia
julia> versioninfo()
Julia Version 1.8.2
Commit 36034abf260 (2022-09-29 15:21 UTC)
Platform Info:
  OS: macOS (arm64-apple-darwin21.3.0)
  CPU: 10 × Apple M1 Max
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-13.0.1 (ORCJIT, apple-m1)
  Threads: 1 on 8 virtual cores
```

**Comment by odow** [comment]
Did you manually install 6.5.2? And did you install the arm version or the intel version?

What is `import Pkg; Pkg.status()`?

**Comment by mzy2240** [comment]
Yes I manually installed 6.5.2, and it is an Arm version which works fine with my previous PR https://github.com/COPT-Public/COPT.jl/pull/21. I think the reason might be, there is only MOI folder for 6.5.3.

**Comment by odow** [comment]
Ah. Yeah it looks like this logic is not correct:
https://github.com/COPT-Public/COPT.jl/blob/f36b08e6b52ae34292c5bfc6e3ac2bb4f97d3c3c/src/COPT.jl#L72-L90

If you've installed 6.5.2, it can't find the 6.5.3 folder.

Here's what we do in Gurobi:
https://github.com/jump-dev/Gurobi.jl/blob/fbe50446d3203dc73ba0be56e3c2f6c13a6056bc/src/Gurobi.jl#L48-L64

**Comment by joachimdahl** [comment]
Maybe the easiest is to generate the missing 6.5.1 and 6.5.2 folders.   They are included in #27.  Does that resolve your issue?

**Comment by mzy2240** [comment]
LGTM
